atd-vzd/migrations/default/1697646059131_changing-keying-on-ldm-view/up.sql

```diff
@@ -0,0 +1,54 @@
+    COALESCE(vz_atd_txdot_person.prsn_bac_test_rslt, cris_atd_txdot_person.prsn_bac_test_rslt) AS prsn_bac_test_rslt,
+    COALESCE(vz_atd_txdot_person.prsn_drg_spec_type_id, cris_atd_txdot_person.prsn_drg_spec_type_id) AS prsn_drg_spec_type_id,
+    COALESCE(vz_atd_txdot_person.prsn_drg_rslt_id, cris_atd_txdot_person.prsn_drg_rslt_id) AS prsn_drg_rslt_id,
+    COALESCE(vz_atd_txdot_person.prsn_taken_to, cris_atd_txdot_person.prsn_taken_to) AS prsn_taken_to,
+    COALESCE(vz_atd_txdot_person.prsn_taken_by, cris_atd_txdot_person.prsn_taken_by) AS prsn_taken_by,
+    COALESCE(vz_atd_txdot_person.prsn_death_date, cris_atd_txdot_person.prsn_death_date) AS prsn_death_date,
+    COALESCE(vz_atd_txdot_person.prsn_death_time, cris_atd_txdot_person.prsn_death_time) AS prsn_death_time,
+    COALESCE(vz_atd_txdot_person.sus_serious_injry_cnt, cris_atd_txdot_person.sus_serious_injry_cnt) AS sus_serious_injry_cnt,
+    COALESCE(vz_atd_txdot_person.nonincap_injry_cnt, cris_atd_txdot_person.nonincap_injry_cnt) AS nonincap_injry_cnt,
+    COALESCE(vz_atd_txdot_person.poss_injry_cnt, cris_atd_txdot_person.poss_injry_cnt) AS poss_injry_cnt,
+    COALESCE(vz_atd_txdot_person.non_injry_cnt, cris_atd_txdot_person.non_injry_cnt) AS non_injry_cnt,
+    COALESCE(vz_atd_txdot_person.unkn_injry_cnt, cris_atd_txdot_person.unkn_injry_cnt) AS unkn_injry_cnt,
+    COALESCE(vz_atd_txdot_person.tot_injry_cnt, cris_atd_txdot_person.tot_injry_cnt) AS tot_injry_cnt,
+    COALESCE(vz_atd_txdot_person.death_cnt, cris_atd_txdot_person.death_cnt) AS death_cnt,
+        CASE
+            WHEN vz_atd_txdot_person.last_update IS NOT NULL AND cris_atd_txdot_person.last_update IS NOT NULL AND vz_atd_txdot_person.last_update > cris_atd_txdot_person.last_update THEN vz_atd_txdot_person.last_update
+            WHEN vz_atd_txdot_person.last_update IS NOT NULL AND cris_atd_txdot_person.last_update IS NOT NULL AND vz_atd_txdot_person.last_update < cris_atd_txdot_person.last_update THEN cris_atd_txdot_person.last_update
+            ELSE COALESCE(vz_atd_txdot_person.last_update, cris_atd_txdot_person.last_update)
+        END AS last_update,
+    COALESCE(vz_atd_txdot_person.updated_by, cris_atd_txdot_person.updated_by) AS updated_by,
+    --COALESCE(vz_atd_txdot_person.person_id, cris_atd_txdot_person.person_id) AS person_id,
+    COALESCE(vz_atd_txdot_person.is_retired, cris_atd_txdot_person.is_retired) AS is_retired,
+    COALESCE(vz_atd_txdot_person.years_of_life_lost, cris_atd_txdot_person.years_of_life_lost) AS years_of_life_lost,
+    vz_atd_txdot_person.person_id as vz_person_id,
+    cris_atd_txdot_person.person_id as cris_person_id
+   FROM vz_facts.atd_txdot_person vz_atd_txdot_person
+     JOIN cris_facts.atd_txdot_person cris_atd_txdot_person ON 
+       vz_atd_txdot_person.crash_id = cris_atd_txdot_person.crash_id AND 
+       vz_atd_txdot_person.unit_nbr = cris_atd_txdot_person.unit_nbr AND 
+       vz_atd_txdot_person.prsn_nbr = cris_atd_txdot_person.prsn_nbr;
```

Comment:
Join the tables on just what we know will be available in both tables, always. These are the actual fields that are used to uniquely identify a person.